plugins/woocommerce/tests/e2e-pw/utils/tours.js

```diff
@@ -0,0 +1,31 @@
+const { admin } = require( '../test-data/data' );
+
+const base64String = Buffer.from(
+	`${ admin.username }:${ admin.password }`
+).toString( 'base64' );
+
+const headers = {
+	Authorization: `Basic ${ base64String }`,
+	cookie: '',
```

Comment:
Out of curiosity, is there any reason why we need to set the `cookie` header empty here?

Reply:
Oh, great catch @alopezari! `cookie` is not necessary in this case. I had just pushed 02638a71e831389bb109766116eca2a4d43ccdc4 to remove it.

I copied that function from another E2E test. That test used the `page` object to construct the `request` object so it needed to have an empty `cookie` header. Otherwise, that test will get a `401 Unauthorized` as a response.

But in this test, it passed even without the `cookie` header so I removed it. I guess it's because the function, particularly in this [line](https://github.com/woocommerce/woocommerce/blob/02638a71e831389bb109766116eca2a4d43ccdc4/plugins/woocommerce/tests/e2e-pw/utils/tours.js#L17), uses the `request` object instead of the usual `page` object.